File: python/nivel 3 listas textos/17.py
"""
Ejercicio 17
Dado un texto, devolver la palabra más larga
"""
texto = input("Ingrese un texto: ")

palabras = texto.split()

if not palabras:
  print("no se ingresaron palabras")
else:
  palabra_mas_larga = palabras[0]

  for palabra in palabras:
    if len(palabra) > len(palabra_mas_larga):
      palabra_mas_larga = palabra

  print(f"la apalbra mas larga es: {palabra_mas_larga}")

File: javascript/nivel 3 listas textos/17.js
/*
Ejercicio 17
Dado un texto, devolver la palabra más larga
*/
let texto = prompt("Ingrese un texto: ");

if(texto === null || texto.trim() === ""){
  console.log("Nose ingresaron palabras");
}else{
  let palabras = texto.split(" ");
  let palabraMasLarga = palabras[0];

  for (let i = 0; i < palabras.length; i++) {
    if(palabras[i].length > palabraMasLarga.length){
      palabraMasLarga = palabras[i];
    }
  }
  console.log("la palabra mas laraga es: ", palabraMasLarga);
}
